shared/unattended/OpenSUSE-Leap-42.2.xml

```diff
@@ -0,0 +1,961 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list"/>
+  </add-on>
+  <bootloader>
+    <device_map config:type="list">
+      <device_map_entry>
+        <firmware>hd0</firmware>
+        <linux>/dev/vda</linux>
```

Comment:
Sorry for the delay, I've been keeping busy recently.

Good catch.
BTW, There is one more section which blocks the installation with virtio-scsi.
I will fix them later, Thanks for your review.